datasets/id_panl_bppt/id_panl_bppt.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""PANL BPPT"""
+
+from __future__ import absolute_import, division, print_function
+
+import logging
+import os
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{id_panl_bppt,
+  author    = {PAN Localization - BPPT},
+  title     = {Parallel Text Corpora, English Indonesian},
+  year      = {2009},
+  url       = {http://digilib.bppt.go.id/sampul/p92-budiono.pdf},
+}
+"""
+
+_DESCRIPTION = """\
+Parallel Text Corpora for Multi-Domain Translation System created by BPPT (Indonesian Agency for the Assessment and
+Application of Technology) for PAN Localization Project (A Regional Initiative to Develop Local Language Computing
+Capacity in Asia). The dataset contains around 24K sentences divided in 4 difference topics (Economic, international,
+Science and Technology and Sport).
+"""
+
+_HOMEPAGE = "http://digilib.bppt.go.id/sampul/p92-budiono.pdf"
+
+_LICENSE = ""
+
+_URLs = ["https://github.com/cahya-wirawan/indonesian-language-models/raw/master/data/BPPTIndToEngCorpusHalfM.zip"]
+
+
+class IdPanlBpptConfig(datasets.BuilderConfig):
+    """BuilderConfig for IdPanlBppt"""
+
+    def __init__(self, src_tag=None, tgt_tag=None, topics=None, **kwargs):
+        """BuilderConfig for IdPanlBppt.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(IdPanlBpptConfig, self).__init__(**kwargs)
+        self.src_tag = src_tag
+        self.tgt_tag = tgt_tag
+        self.topics = topics
+
+
+class IdPanlBppt(datasets.GeneratorBasedBuilder):
+    VERSION = datasets.Version("1.0.0")
+
+    BUILDER_CONFIGS = [
+        IdPanlBpptConfig(
+            name="id_panl_bppt",
+            version=VERSION,
+            description="IdPanlBppt dataset",
+            src_tag="en",
+            tgt_tag="id",
+            topics=[
+                {"name": "Economy", "words": "150K"},
+                {"name": "International", "words": "150K"},
+                {"name": "Science", "words": "100K"},
+                {"name": "Sport", "words": "100K"},
+            ],
+        ),
+    ]
```

Comment:
```suggestion
    ]
    BUILDER_CONFIG_CLASS = IdPanlBpptConfig
```
we need that to specify that we're using a custom config here